docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       - UDW_DATABASE=${UDW_DATABASE}
     entrypoint: ['docker-entrypoint.sh', '--default-authentication-plugin=mysql_native_password']
     ports:
-      - "3306"
+      - "33306:3306"
```

Comment:
I added this because without it the port is randomized. This just makes it easier to connect to from a SQL client on the local host. This won't change the Openshift. 

Like how we have 5001 for the server, having an actual port for the MySQL just makes dev a little easier. This should have been on a different PR tho.